jprov/server/proofs.go

```diff
@@ -310,6 +310,7 @@ func postProofs(cmd *cobra.Command, db *leveldb.DB, q *queue.UploadQueue, ctx *u
 				ctx.Logger.Error(fmt.Sprintf("Posting Proof Error: %v", err))
 				continue
 			}
+			time.Sleep(time.Second)
```

Comment:
Doesn't appear to be working. At this point, I bumped up my gas-prices to 0.09jkl, and I did not get time out errors for over an hour. It's evident that txs with higher fees get priority, but this is obviously not a scalable solution. 

Also, as at current time--2 p.m. ET April 14--all the providers except for mine and Marston's have the default 32 second interval between proofs--likely that most of them are committing proofs at the exact same time and causing timeout errors.